src/attributes.jl

```diff
@@ -162,7 +162,7 @@ canset(instance, ConstraintPrimal(), ConstraintIndex{VectorAffineFunction{Float6
 """
 function canset end
 canset(instance::AbstractInstance, attr::AnyAttribute) = false
-canset(instance::AbstractInstance, attr::AnyAttribute, ref::Index) = false
+canset(instance::AbstractInstance, attr::AnyAttribute, ref::Type{<:Index}) = false
 canset(instance::AbstractInstance, attr::AnyAttribute, refs::Vector{<:Index}) = false
```

Comment:
Should this vector version still exist?

Reply:
I'd vote to remove it

Reply:
Is it even documented?

Reply:
Yes, it is. But anyway it's redundant. Let's remove it.